docs/FAQ/gui-faq-en.md

```diff
@@ -4,163 +4,178 @@ Zonemaster
+in which the references to the standard documents for that test case are found.
+
+The descriptions of message levels such as *notice*, *warning* and *error* are found 
+in [Severity Level Definitions].
+
+Sometimes there are different interpretations of the standards or opinions on what is best practice,
+and the Zonemaster team is always open to input.
+If you think we have made a mistake in our judgement please do not hesitate to send us an email
+at [zonemaster-users@lists.iis.se] (moderated mailing list) with a link to your test result
+and an explanation as to why you think it shows something that you consider incorrect.
 
 <a name="q6"></a>
-#### 6. Does Zonemaster handle IPv6?
-Yes, it does. All tests run over IPv4 will also be run over IPv6 if Zonemaster
-is configured to do so.
+#### 6. Does Zonemaster support IPv6?
+Yes.
+By default Zonemaster will query name servers both over IPv4 and IPv6, unless explicitly
+configured otherwise.
+Such configuration is accessible through the "Options" button.
 
 <a name="q7"></a>
-#### 7. Does Zonemaster handle DNSSEC?
-Yes, if DNSSEC is available for a domain that is tested by Zonemaster, it will be
+#### 7. Does Zonemaster verify DNSSEC?
+Yes.
+If DNSSEC is available for a domain name that is tested by Zonemaster, it will be
 checked automatically.
 
 <a name="q8"></a>
 #### 8. What makes Zonemaster differ from other DNS zone validating software?
 Firstly, Zonemaster saves all history from earlier tests based on the tested
-domain, which means you can go back to a test you did a week ago and compare it
-to the test you ran a moment ago.
+domain name, which means you can go back to a test you did some time ago and compare it
+to the test you ran just a moment ago.
 
-All tests that Zonemaster runs are defined in test case specifications that
-are linked from [Test Requirements
-document](https://github.com/zonemaster/zonemaster/blob/master/docs/requirements/TestRequirements.md)
+Secondly, all tests that Zonemaster runs are defined in Test Case specifications that
+can be found in the [Defined Test Cases] document.
 
-Zonemaster could be used to test undelegated domains. More about undelegated
-domains in [Question 12](#q12).
+Thirdly, Zonemaster can be used to test undelegated domain names.
+See [Question 12].
+
+Fourthly, Zonemaster can be used to test DS records before their publication in the parent zone.
+See [Question 13].
 
 Lastly, this open source version of Zonemaster was built using modular code
-which, basically, means you can use parts of it in your systems, if you would want
-to. It is quite rare that you'd want a complete program just to check for example
+which basically means that you can integrate parts of it in your own systems, if you wish.
+For example, it is quite rare that you would want a complete program just to check for
 redelegations.
 
 <a name="q9"></a>
 #### 9. Zonemaster and privacy
 Since Zonemaster is open to everyone it is possible for anyone to check your
```

Comment:
I think we should refer to zonemaster.net here, since that is what we have under our control. Suggestion
```
Since [Zonemaster.net](https://zonemaster.net/] is open to everyone it is possible for anyone to check your
```

Reply:
Updated